lib/pages/race/moderator/event_statistics.dart

```diff
@@ -1,21 +1,23 @@
 import 'package:flutter/material.dart';
+import 'package:flutter_riverpod/flutter_riverpod.dart';
+import 'package:regatta_buddy/models/round.dart';
+import 'package:regatta_buddy/providers/race_events.dart';
 import 'package:regatta_buddy/utils/constants.dart' as constants;
+import 'package:regatta_buddy/utils/logging/logger_helper.dart';
 
-class EventStatistics extends StatelessWidget {
-  final int round;
+class EventStatistics extends ConsumerWidget {
+  EventStatistics({super.key});
 
-  const EventStatistics(
-    this.round, {
-    super.key,
-    this.height = 50,
-  });
-
-  final double height;
+  final logger = getLogger("EventStatistics");
 
   @override
-  Widget build(BuildContext context) {
+  Widget build(BuildContext context, WidgetRef ref) {
+    final currentRound = ref.watch(currentRoundProvider);
+    final currentRoundStatus = ref.watch(currentRoundStatusProvider);
+
+
     return Container(
-      height: height,
+      height: MediaQuery.of(context).size.height * 0.1,
```

Comment:
Is there a reason we should use media query instead of flex etc?

Reply:
nope, I was just playing with this value